src/main/resources/org/sagebionetworks/web/PortalDebug.gwt.xml

```diff
@@ -2,5 +2,5 @@
 <module rename-to='Portal'>
   <inherits name="org.sagebionetworks.web.Portal" />
 
-  <set-property name="user.agent" value="safari" />
+  <set-property name="user.agent" value="safari,gecko1_8" />
```

Comment:
allow using firefox with PortalDebug module. If you have precompile off, it won't compile it until you actually load the app in devmode with the expected user agent